packages/metadata-tools/src/registry/registry.ts

```diff
@@ -17,18 +19,24 @@ const Registry: IRegistry<IAssetMap> = {
     return this.metadata.components.get(comp)!;
   },
   describeAsset <AssetType extends keyof IAssetMap, Asset extends IAssetMap[AssetType]>(asset: Asset, type: AssetType, name: string, description?: string): IAssetMetadata {
-    // TODO: actually implement
+    if (!this.metadata.assets.has(asset)) {
```

Comment:
Instead of ignoring subsequent attempts to describe an asset I think it should throw an error.